dotCMS/src/main/java/com/dotcms/tika/TikaUtils.java

```diff
@@ -158,6 +161,23 @@
                     "Could not parse file metadata for file : " + binFile.getAbsolutePath() + ". " + ex.getMessage());
             }
         }
+        User systemUser = null;
+        try {
+            systemUser = APILocator.getUserAPI().getSystemUser();
+        } catch (DotDataException e) {
+            Logger.error(this,"Unable to get System User. ",e);
+        }
+
+        Map<String, Object> additionProps = new HashMap<>();
+        try {
+            additionProps = com.dotmarketing.portlets.contentlet.util.ContentletUtil.getContentPrintableMap(systemUser, APILocator.getContentletAPI().find(inode,systemUser,true));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 179 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)